src/libraries/Microsoft.Extensions.Configuration.Binder/README.md

```diff
@@ -89,3 +89,42 @@ You can include a configuration file using a code like this in your `.csproj` fi
   </Content>
 </ItemGroup>
 ```
+
+## Contributing changes to the source generator
```

Comment:
Could you also update the baseline tests so that failure contains a link to this doc?